forecasting/src/autogluon/forecasting/models/gluonts_model/mqcnn/default_parameters.py

```diff
@@ -0,0 +1,11 @@
+def get_default_parameters():
+    """
+    These default values are used for quick test
+    """
+    hps = {
+        "seed": 42,
```

Comment:
add comment to explain each hyperparameter, or provide link to GluonTS docs where user can find explanations

Reply:
this file is no longer in use and I will delete it in next push.

Reply:
and now if user does not enter any parameters, it will by default use gluonts default parameters.